MiddyMorphy: Fix gui init

loowps opened the pull request:
Fix the initial gui:
- Quick (and probably ugly) workaround to restore the change event source in the relevant listener callback
- Also rollback of the LookAndFeel version for now

Discovered crashes on delete of controller and delete of last scene with some of the auto and audit context menu entries enabled. But at least some CC values seem to be spawned.

sleiner reviewed (approved):
I can confirm that the GUI does work now 🙌
Also, I could reproduce both of the bugs that you mentioned 👍🏻 One question though: Did you get meaningful (i.e, not 0-valued) controller messages using the XY pad? I didn't... So either I don't get how this is supposed to work or there is another caveat to list here 😄

loowps commented:
> I can confirm that the GUI does work now 🙌 Also, I could reproduce both of the bugs that you mentioned 👍🏻 One question though: Did you get meaningful (i.e, not 0-valued) controller messages using the XY pad? I didn't... So either I don't get how this is supposed to work or there is another caveat to list here 😄

Yes, I did, but it's a little bit convoluted: 

- I added a Bitwig Midi-Modulator with CC45 as reference,
- unchecked "audit selected scene" in the context menu
- added a third controller and changed all controllers to CC45
- added some more scenes
- selected all scenes one by one (click+ctrl on win) and changed their controller values to random values

I could see the interpolated output of the cc values when moving the xy-cursor then.

loowps commented:
..and thanks for reviewing and testing :)
I also tried to fix the controller delete bug and could pin it down as far as that in case I set the delete object flag of the delete function in the delete callback to false, the crash wouldn't happen. As I implemented a similar plugin myself which is in a more mature state I'm not so motivated to dig deeper (also it's a little bit over my head with all those raw pointers tbh).

sleiner commented:
>> Did you get meaningful (i.e, not 0-valued) controller messages using the XY pad?
>
> Yes, I did, but it's a little bit convoluted

Okay, I see 👍🏻 I have reproduced that, so I'm going to merge this now 👌

> I implemented a similar plugin myself which is in a more mature state

do tell 😊
